### ISSUE TITLE
Slow open on Windows and a decent PC

### ISSUE THREAD

**Issue by gremo** [Eugeny/tabby#2435]
I really love Terminus. 
However, is too much slow on my machine (which is a i9-9900k nearly at 5Ghz, quite a "decent" PC).

I already have:

- [x] Disabled the acrylic background
- [x] Disabled the restore of previous sessions

Any suggestion to improve the loading time?

**Comment by Eugeny** [comment]
How long does it take? You could record a performance trace for a fresh load: https://developers.google.com/web/tools/chrome-devtools/evaluate-performance/reference

- Start Terminus
- Open Devtools from Settings
- Click "reload & record" on the Performance tab: 
![image](https://user-images.githubusercontent.com/161476/79733848-0a06b980-82f6-11ea-889d-bb9cbfdfccf5.png)
- Save the trace: https://developers.google.com/web/tools/chrome-devtools/evaluate-performance/reference#save

**Comment by gremo** [comment]
It took 2912 ms using the performance monitor, doing a full reload when Terminus was already openend.

When it's closed, it's even worst (i.e. launching it from "Open Terminus here".

**Comment by Eugeny** [comment]
Can you upload the trace?

**Comment by gremo** [comment]
[Profile-20200420T110914.zip](https://github.com/Eugeny/terminus/files/4502380/Profile-20200420T110914.zip)

**Comment by Ireristible** [comment]
I'm also facing the same issue on Terminus version 1.0.111
Here are my traces: 
[Trace 1.zip](https://github.com/Eugeny/terminus/files/4605669/Trace.1.zip) - took 5.4 seconds 
[Trace 2.zip](https://github.com/Eugeny/terminus/files/4605670/Trace.2.zip) - took 2.3 seconds 
I ran the traces a few times and the average was 2.3 seconds. Not sure why that first one took that long but I guess you'll be able to tell. Also as @gremo said, it can take much longer to start for the first time. One time it took 15 seconds (manually timed) 😱

I would normally set the Shell profile to "WSL / Ubuntu" but for the trace I set it to "Default (PowerShell Core). Pretty much all other settings are default. No additional plugins or weird configurations. I also already have:
- [x] Disabled the acrylic background
- [x] Disabled the restore of previous sessions

No other apps were open and I checked that there were no running processes slowing down the system. I think my PC is more than capable.

Specs:
_CPU_ - AMD Ryzen 7 3800X @ 3.9GHz
_RAM_ - 32GB DDR4 @ 3600MHz
_Storage_ - 1TB Samsung NVMe M.2 SSD (for OS and everything else. Only 97GB used so far.)
_OS_ - Windows 10 Home, Version 1909, OS build 18363.815 (Installed only 2 weeks ago so hasn't built up "clutter" that could be slowing it down)
